config/locales/models/activity.en.yml

```diff
@@ -83,7 +89,7 @@ en:
         stage: Stage
         status: Status
         third_party_projects: Third-party projects
-        title: '%{level} name'
+        title: 'Title'
```

Comment:
Why has this changed? I have a memory of this being a content change from @duffylouise?

Reply:
Ah I thought it was a bug in a context I saw it being used. When looking at an activity details view for a project the title field was:

```
Third-party project name: <title of project>
```

I will take another look to see if it was in fact because level was being injected incorrectly!

Reply:
My mistake. I added more test coverage to reduce the chances of me accidentally breaking it again https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/pull/515/commits/939f63e8bbdc42a21b27fbe7c296129ee3659bb2